Cargo.toml

```diff
@@ -144,6 +144,6 @@ substrate-wasm-builder = { version = "25.0.0", default-features = false }
 [patch.crates-io]
 sp-state-machine = { git = "https://github.com/Quantus-Network/zk-state-machine" }
 sp-trie = { git = "https://github.com/Quantus-Network/zk-trie" }
-sc-network = { git = "https://github.com/Quantus-Network/sc-network-pqc" }
```

Comment:
I don't think the version gets used here and this is the wrong branch at this point, it should be main after we merge 48.5 in